src/app/components/plugin-uiframe/plugin-uiframe.component.ts

```diff
@@ -273,6 +290,59 @@ export class PluginUiframeComponent implements OnChanges, OnDestroy {
         iframe?.contentWindow?.postMessage?.(message, this.pluginOrigin ?? "*");
     }
 
+    private loadImplementations(): void {
+        const firstPage = this.loadImplementationsFromPage(0);
+        
+        firstPage?.pipe(
+            map(firstPage => {
+                let pages: Observable<ApiObjectList<ExperimentDataApiObject>>[] = [of(firstPage)]
+                for (let i = 1; i < firstPage.itemCount / this.itemsPerPage; i++) {
+                    const page = this.loadImplementationsFromPage(i)
+                    if (page !== null) {
+                        pages.push(page)
+                    }
+                }
+                return pages;
+            }),
+            mergeAll(),
+            map(wholePage => 
+                wholePage.pipe(
+                    map(apiObjectList => apiObjectList.items.filter(experminetData => experminetData.contentType === "application/qasm" || experminetData.contentType === "application/qiskit")),
```

Comment:
Use a set to make the allowed contentTypes configurable and use the `.has` method to check if the content type is allowed.